lib/rails-brotli-cache/store.rb

```diff
@@ -41,16 +41,14 @@ def initialize(core_store, options = {})
     def fetch(name, options = nil, &block)
       options = (options || {}).reverse_merge(@init_options)
 
-      if !block_given? && options[:force]
```

Comment:
Some of the changes on this PR like replacing `block_given?` with `block` etc. is coming from the `standardrb` gem. I can clean those changes up if you want, but because `standardrb` tries to follow the best practices, I wanted to keep them in the PR.